### ISSUE TITLE
Fail during pyglottolog import

### ISSUE THREAD

**Issue by J-Moravec** [D-PLACE/pydplace#5]
Python version: 3.6.7
pydplace version: 2.1.1

Running pydplace I get the following error:
``` 
 File "/home/jmorave/work/dplace/venv/lib/python3.6/site-packages/pydplace/glottolog.py", line 10, in <module>
    from pyglottolog.languoids import Level
ImportError: cannot import name 'Level'
```

Broken due to this commit in pyglottolog removes Level import:
https://github.com/glottolog/pyglottolog/commit/b8118ef93b290b96c8ef9986ebfc050c59ee112b#diff-8006b1f2b73518d0ceb976d9f1f5c163

Seems like that the idea behind that commit is to remove the Level class entirely?

I think as a hotfix, making pydplace dependent on v1.5.1 pyglottolog would solve this problem.

**Comment by xrotwang** [comment]
Ah,  yes. Pinning pyglottolog~=1.5 should work, too.

Jiří Moravec <notifications@github.com> schrieb am Mo., 24. Juni 2019,
07:25:

**Comment by xrotwang** [comment]
Just released a fix with version 2.1.2: https://pypi.org/project/pydplace/
